### ISSUE TITLE
The links of collection name under "Collections of Questions" don't have URI encode.

### ISSUE THREAD

**Issue by thebrecht** [metabase/metabase#5108]
I am using chinese as a collection name, when i click the link of collection name, the collections page will show "not found".  But If I click the collection name from a card page, It works fine. I found the different between two pages is the link under "Collections of Questions"  doesn't have URI encode.

Hoping you can fix it, thanks.

**Comment by camsaul** [comment]
This is certainly weird 

Steps to repro:

1.  Create a new collection titled `TOUCAN IN CHINESE IS 巨嘴鸟`
2.  Add a Card to it
3.  In the Questions view the link is http://localhost:3000/questions/collections/toucan_in_chinese_is_%E5%B7%A8%E5%98%B4%E9%B8%9F
4. When viewing the Card itself the link is
http://localhost:3000/questions/collections/toucan_in_chinese_is_%25E5%25B7%25A8%25E5%2598%25B4%25E9%25B8%259F_

It looks like in both cases the API returns the former, so that seems right.

```json
[
    {
        "id": 66,
        "name": "Toucan in Chinese is 巨嘴鸟 ",
        "slug": "toucan_in_chinese_is_%E5%B7%A8%E5%98%B4%E9%B8%9F_",
        "description": null,
        "color": "#509EE3",
        "archived": false,
        "can_write": true
    }
]
```

It looks like the second one has been URL encoded twice, which is once more than preferable. For what it's worth both links work for me on Google Chrome

@thebrecht what browser are you using?

**Comment by senior** [comment]
@thebrecht bump, do you have more details on this issue?

I can confirm what @camsaul is seeing with single encoding vs. double encoding. Both links work fine for me in Firefox and Chrome. The single encoding looks to be correct. It displays in the browser as unencoded, looking at the HTML source though, it is encoded. The double encoded links still appear to work, but they display as encoded. My guess is the browser decodes the single encoded URLs for display purposes, but doesn't double decode the other URLs.

I'll work on fixing the double encoding issue.

**Comment by mazameli** [comment]
@thebrecht just a heads up that [we've just shipped version 0.26](http://www.metabase.com/blog/Metabase-0.26) which fixes this issue.
